content/kotsadm/installing/stateful-components.md

```diff
@@ -7,69 +7,40 @@ weight: 10065
+This could be the Rook or Minio add-ons.
+Without one of these add-ons, the installer will fail with an error.
 
+Embedded Clusters can have optional, indirect dependencies on the S3-compatible object store. 
+1. The Registry add-on, used for air-gapped, embedded installs, will use object storage if available. Otherwise it will use the default Storage Class.
+1. The Velero add-on, used for snapshots, will default to saving snapshots in the object store (i.e. the **Internal** option in the console).
 
-When using the OCI Registry instead of the S3 compatible object store, a few benefits are realized:
+To deploy an embedded cluster without object storage, set the `disableS3` flag to `true` in the KOTS add-on.
+This will deploy KOTS without object storage, as well as allow the supporting add-ons to use Persistent Volumes (PVs) instead of object storage.
+The behavior of this flag is describe in the [KOTS add-on](https://kurl.sh/docs/add-ons/kotsadm) documentation.
 
-**The OCI registry can be externalized**
+See [Removing Minio](https://kurl.sh/docs/install-with-kurl/removing-minio) for documentation regarding migrating a cluster away from object storage.
 
-The KOTS installation does not need to use it's own, locally configured registry for storage. 
-A customer can choose to provide their own registry to use as storage.
-At this time, KOTS has been tested to be compatible with the following registries:
+## Existing Clusters
 
-- [Docker Hub](https://hub.docker.com)
-- Locally Hosted Docker Registry [(docker distribution)](https://github.com/docker/distribution)
-- [Google Artifact Registry](https://cloud.google.com/artifact-registry/docs/transition-from-gcr)
-- Amazon Elastic Container Registry [(coming soon)](https://github.com/aws/containers-roadmap/issues/308)
-
-**Improved performance**
-
-Because application artifacts are made from layered container images and application manifests, there's similarity between versions.
-Using the OCI format enables KOTS to build on the layer functionality to minimize the storage and bandwidth for each version.
-
-#### Enabling OCI Registry Support
-
-To install KOTS with the built-in OCI registry instead of Minio, run:
-
-```shell
-kubectl kots install --with-dockerdistribution
-```
-
-To install KOTS using your own registry, run:
+By default, a KOTS installation to an existing cluster will deploy Minio to satisfy the object storage requirement, and nothing further is required during installation.
+When deploying, Minio is configured with a randomly generated AccessKeyID and SecretAccessKey, and only exposed as a ClusterIP on the overlay network.
 
-```shell
-kubectl kots install --storage-base-uri docker://docker.io/myorg/myapp
-```
+When the [install command](https://kots.io/kots-cli/install/) flag `--with-minio=false` is used, the installer will not deploy Minio.
+KOTS will deployed as a Statefulset with attached PV instead of a Deployment.
```

Comment:
```suggestion
KOTS will deploy as a Statefulset with attached PV instead of a Deployment.
```

Reply:
`Deployment` and `Statefulset` are both kubernetes types - I guess they should both be capitalized and monospace?